### DISCUSSION TITLE
Ошибка 500 в service.php

### DISCUSSION THREAD

**Discussion by Netoniel** [cdek-it/widget#73] [General]
Берем [ service.php](https://github.com/cdek-it/widget/blob/main/dist/service.php?plain=1) вставляем новые логин и pass. Копируем на сервер в любое место и заходим на него. Получаем ошибку 500. Что не так с кодом?

**Comment by vermorag** [comment]
Добрый день. Для диагностики вашей проблемы, необходимы логи с сервера. Также вы можете обратиться на integrator@cdek.ru с вашим вопросом для более оперативной помощи.
